Lib/zoneinfo/_tzpath.py

```diff
@@ -111,14 +111,14 @@ def available_timezones():
         determine if a given file on the time zone search path is to open it
         and check for the "magic string" at the beginning.
     """
-    from importlib import resources
+    from importlib.resources import files
```

Comment:
I think we should still access `resources` via the top-level name.

I've started doing this more often and I like that it exposes where things are coming from, and in the backport we tend to have some conditional imports like:

```python
try:
    from importlib import resources
except ImportError:
    import importlib_resources as resources
```

I *think* it's easier to maintain that sort of thing when the conditionally-imported namespace is a single line (though in practice we may have to switch the backport over to have a `_resources_compat` that conditionally defines a package exposing something like the `files` API).

Reply:
I agree, and in all cases except `files`, I've followed that approach. In usage of `importlib resources`, I've deviated from that approach because `resources.files` consumes a lot of characters on the line for a line that _often_ has a lot more syntax (package name, joinpath, action). In this case, the line fits nicely either way, so I'm happy to follow your instincts on it.

Reply:
```suggestion
    from importlib import resources
```